cro-clib/Cargo.toml

```diff
@@ -6,7 +6,8 @@ edition = "2018"
 
 [lib]
 name = "cro_clib"
-crate-type = ["staticlib"]
+#in ios, android, use staticlib instead of cdylib
+crate-type = ["cdylib"]
```

Comment:
isn't staticlib better default?

Reply:
for staticlib, link is OK, but when i run example on ubuntu, its crashes (seg-fault)

Reply:
where does it segfault? can you do strace of what's the last thing it calls?

Reply:
or run it in a debugger?

Reply:
i tried again, it's working fine with ["staticlib"], i'll change back